src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,78 @@
+# Genetic Descriptor
+
+Support for genetic descriptors was developed as a
+[BIDS Extension Proposal](https://github.com/bids-standard/bids-specification/blob/master/src/06-extensions.md#bids-extension-proposals).
+The extension was primarily developped by Cyril Pernet and Clara Moreau
+with contributions from Tom Nichols and Jessica Turner.
+
+The goal of the genetic descriptor is to link imaging and genetic data. This is necessary as genetic data are typically stored in dedicated repositories, separately from the imaging data. The descriptor provides basics information about:
+-   where to find genetic information associated with the imaging data
```

Comment:
No directory structure is specified for genetics data in this PR? Top-level [genetics] seems most intuitive.

Reply:
indeed, as explained the goal is to indicate where to find genetic information associated with the imaging data and what type of genetic information is available

Reply:
indeed, as explained the goal is to indicate where to find genetic information associated with the imaging data and what type of genetic information is available -- so really not much but some metadata

Reply:
Sorry, I'm relatively new to this, but isn't that a problem? Knowing where to get access to the data (dbGaP, phs001364.v1.p1) is useful for people, but less so for tools. If I'm using code to perform some analysis, the only data available to me through BIDS would be in the participants.tsv file (e.g. "IDH mutation" from line 59), which would need to be regenerated for different variants. In cases with large datasets and users with read-only access, that seems like a serious limitation.
Given that the participants.tsv + genetic_info.json files give a lot of information that is both user- and machine-readable (and often not as readily-available as proposed here), it seems to me that adding a directory structure spec would be a huge improvement.

If the intent is to provide user-readable metadata, then I agree that specifying a directory structure is beyond the scope of this BEP (in which case you can mark this as resolved).

Reply:
well yes it is - but there no way this can happen ; genetic data are protected by ethic governance because they are personal data and this cannot be public the way we can for imaging == >> can't do what you are asking for in 99% of cases - so yeh we'll stick to user-readable metadata

Reply:
Isn't sharing the data a separate issue? The data doesn't need to be public for it to be BIDS-compliant. If somebody has collected imaging + genetics data, they'll have access to the data and the question of ethics has already been addressed. BIDS specifies how to structure imaging data regardless of whether the data is shared (Nifti format, directories for anat & func, etc.), but the genetics BEP doesn't specify how to structure genetics data (e.g. data format: text and/or binary PLINK, filenames for different chromosomes, imputed & derived data, etc.).
Since participants.tsv provides the imaging <-> genetics link, the standard implicitly assumes that the genetics data is available while not specifying how it is supposed to be formatted.

Reply:
that's just not what we made it for -- now once merged nothing say you can work on version 2